### ISSUE TITLE
Download correct version of Gesso

### ISSUE THREAD

**Issue by mortona42** [forumone/generator-web-starter-gesso#20]
WordPress: https://github.com/forumone/gesso-wp/
D8: https://github.com/forumone/gesso
D7: https://github.com/forumone/gesso/tree/7.x-2.x

**Comment by mortona42** [comment]
Here's what we've got currently:

```javascript
case 'wordpress':
            promise = this.remoteAsync('forumone', 'gesso-wp', 'master');
            break;

          case 'drupal':
            promise = drupal_modules.getLatestMinorVersions('gesso')
              .then(function(versions) {
                // If the user selected Libsass use the most recent release
                // Otherwise use the most recent release of 7.x-1.x
                var url = (SASS_CHOICES[0] === that.config.get('sass')) ?
                    _.find(versions, { version_major : 7 }).download_link :
                    _.find(versions, { version_major : 7, version_minor : 1 }).download_link;

                return that.remoteAsync(url);
              });
            break;

          case 'drupal8':
            promise = drupal_modules.getVersions('gesso')
              .then(function(versions) {
                var url = _.find(versions, { version_major : 8, version_minor : 1 }).download_link;
                return that.remoteAsync(url, true);
              });
            break;
```

**Comment by mortona42** [comment]
Looks like WP is downloading the latest from the gesso-wp repo.

Simplifying everything else to do the same:

```javascript

          case 'wordpress':
            promise = this.remoteAsync('forumone', 'gesso-wp', 'master');
            break;

          case 'drupal':
            promise = this.remoteAsync('forumone', 'gesso', '7.x-2.x');
            break;

          case 'drupal8':
            promise = this.remoteAsync('forumone', 'gesso', 'master');
            break;

```

**Comment by mortona42** [comment]
Made the change, checkout my generator-web-starter-gesso repo to test:  https://github.com/mortona42/generator-web-starter-gesso

Use master branch for all latest changes, or check out a specific branch.

Instructions for testing my repo can be found: https://forumone.atlassian.net/wiki/pages/resumedraft.action?draftId=42272504&draftShareId=dc9e167b-9ad4-4173-af23-a28924543172

**Comment by wwhurley** [comment]
Resolved as part of 0.2
